lib/node_modules/@stdlib/blas/base/drot/README.md

```diff
@@ -0,0 +1,204 @@
+// x => <Float64Array>[ ~4.4, ~5.8, 7.2, 8.6, 10.0 ]
+// y => <Float64Array>[ ~4.2, 4.4, 4.6, 4.8, 5.0 ]
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **x**: first input [`Float64Array`][mdn-float64array].
+-   **strideX**: index increment for `x`.
+-   **y**: second input [`Float64Array`][mdn-float64array].
+-   **strideY**: index increment for `y`.
+-   **c**: cosine of the angle of rotation.
+-   **s**: sine of the angle of rotation.
+
+The `N` and stride parameters determine how values in the strided arrays are accessed at runtime. For example, to apply a plane rotation to every other element,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+drot( x.length, x, 2, y, 2, 0.707, 0.707 );
+// x => <Float64Array>[ ~5.7, 2.0, ~8.5, 4.0, ~11.3, 6.0 ]
+// y => <Float64Array>[ ~4.2, 8.0, ~4.2, 10.0, ~4.2, 12.0 ]
+```
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var x0 = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y0 = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+// Create offset views...
+var x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var y1 = new Float64Array( y0.buffer, y0.BYTES_PER_ELEMENT*3 ); // start at 4th element
+
+drot( 3, x1, -2, y1, 1, 0.8, 0.6 );
+// x0 => <Float64Array>[ 1.0, ~8.8, 3.0, 9.8, 5.0, 10.8 ]
+// y0 => <Float64Array>[ 7.0, 8.0, 9.0, 4.4, 6.4, ~8.4 ]
+```
+
+#### drot.ndarray( N, x, strideX, offsetX, y, strideY, offsetY, c, s )
+
+Applies a plane rotation using alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+
+drot.ndarray( x.length, x, 1, 1, y, 1, 1, 0.8, 0.6 );
+// x => <Float64Array>[ 1.0, ~5.8, 7.2, 8.6, 10.0 ]
+// y => <Float64Array>[ 6.0, 4.4, ~4.6, ~4.8, 5.0 ]
+```
+
+The function has the following additional parameters:
+
+-   **offsetX**: starting index for `x`.
+-   **offsetY**: starting index for `y`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example, to apply a plane rotation to every other element starting from second element,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+drot.ndarray( x.length, x, 2, 2, y, 2, 2, 0.8, 0.6 );
```

Comment:
```suggestion
drot.ndarray( 3, x, 2, 1, y, 2, 1, 0.8, 0.6 );
```

Example return values need to be updated. Again, `N` cannot be `x.length` based on the parameterization. Furthermore, the text above says "starting from the second element", and JS uses zero-based indexing.

Reply:
Sure, Athan, I will review all the examples once and update them accordingly..

Reply:
Thanks!